src/Menu/Menu.js

```diff
@@ -396,10 +400,27 @@ class Menu extends Component {
         event.preventDefault();
         this.decrementKeyboardFocusIndex();
         break;
+      default:
+        const hotKeys = this.hotKeyHolder.append(key);
```

Comment:
Right. Now how we detect that. `if (key.length === 1)` is the best I think. Using [`keycode.code`](https://github.com/timoxley/keycode/blob/master/index.js#L45) to exclude can be also an option, but in that case some symbols (`[` and `-` etc.) will go away.